front-end/cypress/integration/sprint-6-test-110.spec.ts

```diff
@@ -0,0 +1,124 @@
+// @ts-check
+
+import { before } from 'lodash';
+import { isContext } from 'vm';
+import { GenerateContactObject, EnterContact } from '../support/contacts.spec';
+
+/*
+
+        Individual
+
+*/
+
+var contact_type;
+var contacts = { Individual: {}, Candidate: {}, Committee: {}, Organization: {} };
```

Comment:
let's change "var" where ever it is found to "let" if the variable is changed and "const" if the variable is never changed or should not be changed